### ISSUE TITLE
viki.com Download Error

### ISSUE THREAD

**Issue by dkl3** [rg3/youtube-dl#14306]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.09.15*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x ] I've **verified** and **I assure** that I'm running youtube-dl **2017.09.15**

### Before submitting an *issue* make sure you have:
- [x ] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---
~⟫ youtube-dl  https://www.viki.com/videos/44699v-boys-over-flowers-episode-1 -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.viki.com/videos/44699v-boys-over-flowers-episode-1', '-v']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.09.15
[debug] Python version 3.5.2 - Linux-4.4.0-93-generic-x86_64-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 2.8.11-0ubuntu0.16.04.1, avprobe 2.8.11-0ubuntu0.16.04.1, ffmpeg 2.8.11-0ubuntu0.16.04.1, ffprobe 2.8.11-0ubuntu0.16.04.1, rtmpdump 2.4
[debug] Proxy map: {}
[viki] 44699v: Downloading video JSON
[viki] 44699v: Downloading video streams JSON
ERROR: viki returned error: invalid signature
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/YoutubeDL.py", line 776, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/common.py", line 434, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/viki.py", line 270, in _real_extract
    'Downloading video streams JSON')
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/viki.py", line 69, in _call_api
    self._raise_error(resp['error'])
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/viki.py", line 76, in _raise_error
    expected=True)
youtube_dl.utils.ExtractorError: viki returned error: invalid signature

---

Basically, scraping www.viki.com doesn't work. I don't think anybody else has reported this yet?
